.gitignore

```diff
@@ -49,3 +49,4 @@ Session.vim
 .vscode
 /www/test_out
 .*.timestamp
+/docs
```

Comment:
Should this not be `docs` or `docs/`?
Either that or TIL a new `.gitignore` syntax. So many of them

Reply:
My understanding is that this ignores a docs directory in the root of the project but would not ignore something like `foo/docs` in the future. We already have a surprising amount of lines in our `.gitignore` with the `/` prefix so hopefully it's not too out of place.